browser/ui/webui/settings/brave_settings_clear_rewards_data_handler.h

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_SETTINGS_CLEAR_REWARDS_DATA_HANDLER_H_  // NOLINT
+#define BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_SETTINGS_CLEAR_REWARDS_DATA_HANDLER_H_  // NOLINT
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/containers/flat_set.h"
+#include "base/macros.h"
+#include "base/observer_list_types.h"
+#include "base/scoped_observer.h"
+#include "brave/components/brave_rewards/browser/rewards_service_observer.h"
+#include "brave/components/brave_rewards/browser/rewards_service.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/webui/settings/settings_page_ui_handler.h"
+#include "components/browsing_data/core/counters/browsing_data_counter.h"
+
+namespace base {
+class ListValue;
+}
+
+namespace content {
+class WebUI;
+}
+
+namespace settings {
+
+enum class RewardsDataType {
```

Comment:
isn't this a duplicate of BraveRewardsDataRemoverDelegate::DataType? Why have another enum here?

Reply:
No. This is the type that is used to compare against with `BraveRewardsDataRemoverDelegate::DataType`

`RewardsDataType` is created from values sent from the UI